javascript/example-customer-account--pre-auth--ts/extensions/customer-account-pre-auth-note/shopify.extension.toml

```diff
@@ -0,0 +1,21 @@
+api_version = "unstable"
+
+[[extensions]]
+type = "ui_extension"
+name = "customer-account-pre-auth-note-js"
+handle = "pre-auth-note-js"
+
+# [START config.setup-targets]
+[[extensions.targeting]]
+module = "./src/MenuActionItemButtonExtension.ts"
+target = "customer-account.order.action.menu-item.render"
+
+[[extensions.targeting]]
+module = "./src/MenuActionModalExtension.ts"
+target = "customer-account.order.action.render"
+# [END config.setup-targets]
+
+# [START config.api-access]
+[extensions.capabilities]
+# api_access = true
+# [END config.api-access]
```

Comment:
Similar comment here, I don't think this is needed?

Reply:
updated~